apps/neoscan_web/lib/neoscan_web/views/common_view.ex

```diff
@@ -97,6 +98,19 @@ defmodule NeoscanWeb.CommonView do
 
   def render_date_time(date_time), do: DateTime.to_unix(date_time)
 
+  def render_asset_name(conn, asset) do
+    lang = Conn.get_session(conn, "locale")
+
+    specific_name =
+      Enum.find(asset.name, fn x -> x["lang"] == lang end) ||
+        Enum.find(asset.name, fn x -> String.starts_with?(x["lang"], lang) end) ||
```

Comment:
@yongjiema can you add test coverage for this part? Also I am not sure of the String.starts_with? logic you expect locale to be "en" while "lang" is "en-US" ? Are you sure it is the case?